gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/EntityServiceAndGatewayServiceConverter.java

```diff
@@ -165,21 +164,7 @@ private static Function convertToQueryFunction(
             .setFunctionName(function.getFunction().name())
             .setAlias(function.getAlias());
 
-    // AVGRATE is adding a specific implementation because Pinot does not directly support this
-    // function
     switch (function.getFunction()) {
-      case AVGRATE:
```

Comment:
At gateway service layer, the AVGRAGE argument accept literal for now.
```
function {
    function: AVGRATE
    arguments {
      columnIdentifier {
        columnName: "BACKEND.errorCount"
      }
    }
    arguments {
      literal {
        value {
          valueType: LONG
          long: 1
        }
      }
    }
```
At QS will now take it as isoDurationString as an argument. So, the argument conversion will require. Or, we will have to change the client at GraphQL layer.